src/app/Core/ExportServices/DataExportServices/generateDataString.service.ts

```diff
@@ -0,0 +1,29 @@
+import { MainFilterLabelInterface } from "../../../Shared/Model/Export.model";
+import {
+    DbCollection,
+    DbKey,
+} from "../../../Shared/Model/Client_Isolate.model";
+import { generateDataCsvString } from "./generateDataCsvString.service";
+
+/**
+ * @desc Returns the table header and the filtered dataset as string to save it as csv
+ * @param {string[]} keys - keys of the dataset
+ * @param {Record<string, string>[]} data - dataset
+ * @returns {string} - dataset as string
+ */
+export function generateDataString(

```

Comment:
this also generates a csv string. Perhaps this could be generateTableCsvString

Reply:
Done. Commit: Rename "generate...CsvString.service"